setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup, find_packages
+
+PACKAGENAME = "testingframework"
+DESCRIPTION = "Module for testing various interatomic potentials"
+AUTHOR = "Libatoms"
+AUTHOR_EMAIL = ""
+
+version = {}
+with open("version.py") as fp:
+    exec(fp.read(), version)
+
+setup(
+    name=PACKAGENAME,
+    packages=find_packages(),
+    version=version["__version__"],
+    description=DESCRIPTION,
+    long_description=open("README.md").read(),
+    install_requires=["scipy", "numpy", "matplotlib", "pandas>=0.21.0", "scipy",],
```

Comment:
- scipy is duplicated
- ase is not there
- pandas is only used in the demo notebook, not the package
- anything else?

there are optional dependencies, that I *wouldn't want to install* unless needed, like `torch`, `torchani`, `pyjulip`, etc.